echo_handlers.go

```diff
@@ -319,45 +333,30 @@ func (svc *Service) AppsCreateHandler(c echo.Context) error {
 			return err
 		}
 
-		// PAY_INVOICE_METHOD permissions
-		if maxAmount > 0 || !expiresAt.IsZero() {
+		requestMethods := c.FormValue("RequestMethods")
+		if requestMethods == "" {
+			return fmt.Errorf("Won't create an app without request methods.")
```

Comment:
Currently this goes to the connections page if you uncheck all request methods. Maybe should have a frontend form validation error. @im-adithya what do you think?